src/main/java/com/marginallyclever/makelangelo/plotter/plotterRenderer/DrawingAssistant.java

```diff
@@ -0,0 +1,115 @@
+package com.marginallyclever.makelangelo.plotter.plotterRenderer;
+
+import com.jogamp.opengl.GL2;
+import com.jogamp.opengl.util.texture.Texture;
+import com.jogamp.opengl.util.texture.TextureIO;
+import com.marginallyclever.convenience.FileAccess;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.BufferedInputStream;
+import java.io.IOException;
+
+public class DrawingAssistant {
```

Comment:
this should probably be part of com.marginallyclevver.convenience.OpenGLHelper or maybe com.marginallyclevver.convenience.DrawingHelper

Reply:
ok moved to `com.marginallyclevver.convenience.DrawingHelper`